content/en/guide/v10/forms.md

```diff
@@ -5,37 +5,39 @@ description: 'How to build awesome forms in Preact that work anywhere.'
 
 # Forms
 
-Forms in Preact work much the same as they do in HTML. You render a control, and attach an event listener to it.
```

Comment:
hey @38elements - I tried to rephrase this so that the title ends up being a single line. Let me know what you think!

Reply:
Thank you for review.

I think "Forms in Preact are largely the same as HTML forms.
The main difference between both is that in most cases the value of input control is controlled by whether DOM node or Component.	Much like in HTML forms, forms in Preact both work by rendering input elements and listening for events triggered as the form is modified. " is better.

I think "The main difference is that in most cases the input controll's value is not controlled by the DOM node, but by component." is better. 

"most cases the value" => "most cases the input controll's value"
"but by Preact" => "but by component"

This explicitly describes the difference in value management between DOM node and component. I think this easily lead to the next sentence.

Reply:
There are some grammar issues in the first one, but I agree it's worth comparing to the DOM version.
What about this:

>Forms in Preact are largely the same as HTML forms.
>
> Like in HTML, forms in Preact work by rendering input elements and listening for events triggered as the form is modified. The main difference when rendering with Preact is that the value of a form input is generally controlled by a Component, rather than by the DOM element itself.

Reply:
I think your suggestion is better.